app/controllers/sessions_controller.rb

```diff
@@ -7,8 +12,20 @@ def create
     redirect_to "http://vm344a.se.rit.edu/", :notice => "Signed in!"
   end
 
+  # destroy a session
   def destroy
     session[:user_id] = nil
     redirect_to "http://vm344a.se.rit.edu/", :notice => "Signed out!"
   end
+
+  # verify the user is authenticated
+  def verify_auth
+    # attempt to access the user based on auth["provider"] and auth["uid"]
+    user = User.find_by_provider_and_uid(auth["provider"], auth["uid"])
+    if user
+      render json: {authenticated: true}
+    else
+      render json: {authenticated: true}
```

Comment:
I disagree. `auth` being true seems more like you're saying "this requires authentication," and while `authed` conveys "you've been authenticated" it's still not immediately obvious.